projects/1.20/assets/more-bows-cj/morebows/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+	"item.morebows.copper_bow": "铜弓",
+	"item.morebows.iron_bow": "铁弓",
+	"item.morebows.golden_bow": "金弓",
+	"item.morebows.emerald_bow": "绿宝石弓",
+	"item.morebows.diamond_bow": "钻石弓",
+	"item.morebows.netherite_bow": "下界合金弓",
+	"item.morebows.gilded_netherite_bow": "镶金下界合金弓",
+	"item.morebows.enderite_bow": "末影合金弓",
+	"item.morebows.gilded_enderite_bow": "镶金末影合金弓",
+
+	"upgrade.morebows.bow_upgrade": "升级弓",
```

```suggestion
	"upgrade.morebows.bow_upgrade": "弓升级",
```
这个在游戏中是在锻造模板之类的说明文本里吗